src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -91,12 +101,17 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
   if (isLoading || !projects) return <LoadingIndicator />;
   if (isError) return <ErrorPage message={error?.message} />;
 
+  const projectOptions: { label: string; id: string }[] = [];
   const wbsDropdownOptions: { label: string; id: string }[] = [];
   projects.forEach((project: Project) => {
     wbsDropdownOptions.push({
       label: `${wbsPipe(project.wbsNum)} - ${project.name}`,
       id: wbsPipe(project.wbsNum)
     });
+    projectOptions.push({
+      label: `${wbsPipe(project.wbsNum)} - ${project.name}`,
```

Comment:
this pipe to the label gets used three times, may as well make it a separate function or even a separate pipe like wbsAndNamePipe(project) or something